cli/src/commands/compile.ts

```diff
@@ -0,0 +1,55 @@
+import { execSync } from "child_process";
+import fs from "fs";
+import path from 'path';
+import { generateDispatcher, generateFile, generateMetadata } from '../../utils/src';
+import { Constants } from "../constants";
+import { Utils } from "../utils";
+
+export class Compile {
```

Comment:
nit: you do not need to have `_` prefixes. You can specify the methods as `private`